src/main/java/com/minecolonies/coremod/client/gui/modules/WindowHutCrafterTaskModule.java

```diff
@@ -48,7 +48,7 @@ public void onOpened()
         super.onOpened();
         final List<IToken<?>> tasks = new ArrayList<>();
 
-        for (final int citizenId : ((IBuildingWorkerView) buildingView).getWorkerId())
+        for (final int citizenId : buildingView.getModuleView(WorkerBuildingModuleView.class).getWorkerId())
```

Comment:
Not sure I understood what you meant here.

Reply:
There's a mixture of calls to `getModuleView`, which can only return exactly one module, and `getModuleViews`, which can return zero or more modules.  It's suspicious to use both on the same module type, because they're contradictory.     It's not _necessarily_ a bug, especially if there can only be zero or one (and not more than one), but it's still potentially inconsistent.

Reply:
True, let me make this more consistent.